src/components/menus/BottomMenu.js

```diff
@@ -83,37 +70,39 @@ export const BottomMenu = ({ menu }) => {
             </ul>
           </div>
         </div>
-        <div className={sty.ContactColumn}>
-            <div className={sty.MenuLinkTitle}> Caymans Contact</div>
-            <div className={sty.info}>
-              <p>{menu.caymans_email}</p>
-            </div>
-            <div className={sty.info}>
-              <p>{menu.caymans_phone}</p>
-            </div>
-            {/* <div className={sty.info}>
-              <p className={sty.address}>
-                {menu.caymans_address}
-              </p>
-            </div> */}
-
-            <div className={sty.MenuLinkTitle}> Turks & Caicos Contact</div>
-            <div className={sty.info}>
-              <p>{menu.turks_and_caicos_email}</p>
-            </div>
-            <div className={sty.info}>
-              <p>{menu.turks_and_caicos_phone}</p>
+      </Container>
+      <div className={sty.bottomBar}>
```

Comment:
Another sort of style/code cleanliness thing thing, this seems like a lot of containers in this section, might be worth double checking that you need them all. All good if you do but it's possible you don't need the divs around the paragraphs here so streamline it a bit you know?